src/components/CheckboxInput/CheckboxInput.story.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withInfo } from '@storybook/addon-info';
+import { action } from '@storybook/addon-actions';
+
+import withTests from '../../util/withTests';
+import State from '../State';
+import CheckboxInput from './CheckboxInput';
+
+storiesOf('CheckboxInput', module)
+  .addDecorator(withTests('CheckboxInput'))
+  .add(
+    'Default CheckboxInput',
+    withInfo()(() => (
+      <State
```

Comment:
Just also added this to my Toggle. I think we need to write some more abstractions like the ones PowerPlug has.